fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1755,18 +1759,28 @@ void ReadYourWritesTransaction::atomicOp( const KeyRef& key, const ValueRef& ope
 }
 
 void ReadYourWritesTransaction::set( const KeyRef& key, const ValueRef& value ) {
-	if (key == LiteralStringRef("\xff\xff/reboot_worker")){
-		BinaryReader::fromStringRef<ClientWorkerInterface>(value, IncludeVersion()).reboot.send( RebootRequest() );
-		return;
-	}
-	if (key == LiteralStringRef("\xff\xff/reboot_and_check_worker")){
-		BinaryReader::fromStringRef<ClientWorkerInterface>(value, IncludeVersion()).reboot.send( RebootRequest(false, true) );
-		return;
-	}
+
 	if (key == metadataVersionKey) {
 		throw client_invalid_operation();
 	}
 
+	if (specialKeys.contains(key)) {
+		if (getDatabase()->apiVersionAtLeast(700)) {
+			return getDatabase()->specialKeySpace->set(this, key, value);
+		} else {
+			if (key == LiteralStringRef("\xff\xff/reboot_worker")) {
```

Comment:
I think this will break the kill command in fdbcli. We should add a note and maybe an issue about it. I'd like to get this PR merged soon though since it's getting big.

Reply:
I add an issue for it [3385](https://github.com/apple/foundationdb/issues/3585)

Reply:
Add workaround code to not break usage of `\xff\xff/reboot_worker` in `7.0`